protocol/proto/authentication.proto

```diff
@@ -24,6 +24,8 @@ enum AuthenticationType {
     AUTHENTICATION_STORED_FACEBOOK_CREDENTIALS = 0x2;
     AUTHENTICATION_SPOTIFY_TOKEN = 0x3;
     AUTHENTICATION_FACEBOOK_TOKEN = 0x4;
+    // this type is returned when requesting AUTHENTICATION_SPOTIFY_TOKEN
+    AUTHENTICATION_UNKNOWN = 0x7;
```

Comment:
Have you had a look at the proto definitions from the official client to see if there is an actual definition for the `0x7` response? I'm hesitant to just slap an unknown definition in, as it is messy and not very readable/maintainable long term if they start to stack up. I know we have (had) a couple in the library, but we generally try to replace them with the actual value from the client once it is available.

Reply:
Where can I find the official definitions? I was thot aware that they are public.

Reply:
@sashahilton00 I think that those definition files cannot be extracted from the client anymore for some reason. They don't appear when dumping with [HearthSim/proto-extractor](https://github.com/HearthSim/proto-extractor).

Reply:
I believe it should be along these lines - dumped from the client strings. 
Also to note, I no longer see the `kAuthenticationType` mentioned anywhere, and only `kAuthenticationTypeCount` 
`credentials.auth_type() < auth::kAuthenticationTypeCount:`
```
obfuscatedSecret
password
blobBase64
autologin
storedSpotifyCredentials
storedFacebookCredentials
spotifyToken
facebookToken
anonymousAccount
storedAnonymousCredentials
storedSpotifyRefreshToken
spotifyPassword
phoneNumber
oneTimeToken
```